analysis/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM centos:7
```

Comment:
Please add standard copyright header here. You can use a tool I wrote for this: https://github.com/mbrukman/autogen :-)

Reply:
Isn't this already using the standard copyright header?

Reply:
@pluradj The copyright header wasn't there originally when the comment was added.

Reply:
ah ok. I would have expected the comment to be collapsed with `Show outdated`. thanks.